frontend/src/app/settings/core/pipelines-overview/pipelines-overview.component.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { Component } from '@angular/core';
+import { PipelineRunStatus } from 'src/app/projects/models/backup-settings/pipeline-runs/service/pipeline-run.service';
+import { environment } from 'src/environments/environment';
+
+@Component({
+  selector: 'app-pipelines-overview',
+  templateUrl: './pipelines-overview.component.html',
+  styleUrls: ['./pipelines-overview.component.css'],
+})
+export class PipelinesOverviewComponent {
+  constructor(private httpClient: HttpClient) {
+    this.fetchModelHealth();
+    this.fetchProjectHealth();
+    this.fetcGeneralHealth();
+  }
```

Comment:
First of all, I would prefer having this in the `ngOnInit` rather than here in the constructor.

Then is there any reason why you did not create a `pipeline-overview.service.ts`? I assume that it is because then there wouldn't be much in this component but I would still go for it. Especially when you use three behaviour subjects there, you can just use async pipes in the `pipelines-overview.component.html` and you just have to call the `this.pipelineOverviewService.loadModelHealth()` functions in the ngOnInint.

Reply:
I wanted to keep it as simple as possible and not another abstraction layer. But I'll add a service for consistency.

Reply:
However, I have not added a complex BehaviourStructure for it. That's too complex, I'd say.

Reply:
That is the result: 

```ts
  toolmodelStatuses?: ToolmodelStatus[];
  projectStatuses?: ProjectStatus[];
  generalHealth?: GeneralHealth;

  ngOnInit(): void {
    this.monitoringService
      .fetchGeneralHealth()
      .subscribe((generalHealth) => (this.generalHealth = generalHealth));

    this.monitoringService
      .fetchProjectHealth()
      .subscribe((projectStatuses) => (this.projectStatuses = projectStatuses));
    this.monitoringService
      .fetchModelHealth()
      .subscribe(
        (toolmodelStatuses) => (this.toolmodelStatuses = toolmodelStatuses)
      );
  }

```